pkg/config/lpe_windows.go

```diff
@@ -0,0 +1,93 @@
+//go:build windows
```

Comment:
nit: I don't believe these lines are necessary if the file is named *_windows.go.

Reply:
Removed them. I'm still trying to make a better understanding of file suffixes, build tags and their differences by go version and how everything works together. At least make was successful for all targets locally.